src/utl/test/test_report_metrics.tcl

```diff
@@ -0,0 +1,6 @@
+read_liberty ./../../../test/Nangate45/Nangate45_fast.lib
+read_lef ./../../../test/Nangate45/Nangate45.lef
+read_verilog ./../../../test/gcd_nangate45.v
+link_design gcd
+read_sdc ./../../../test/timing_api_2.sdc
+report_metrics 2 final
```

Comment:
this really should be `report_metrics -stage 2 -when final`? @maliberty that feels somewhat awkward to have a numbering scheme in OpenROAD.
`-stage` and `-when` is really just a name for the report, so maybe that should be renamed/simplified?